app/scripts/dimApp.config.js

```diff
@@ -24,6 +24,8 @@
       debugMode: false,
       // show changelog toaster
       changelogToaster: '$DIM_FLAVOR' === 'release' || '$DIM_FLAVOR' === 'beta',
+
+      materialsExchangeEnabled: 'DIM_FLAVOR' === 'beta',
```

Comment:
You could also do `'$DIM_FLAVOR' !== 'release'` so it's still on in local development too.